flow/project.py

```diff
@@ -549,7 +575,7 @@ def __init__(self, cmd, pre=None, post=None):
     def __str__(self):
         return "{type}(cmd='{cmd}')".format(type=type(self).__name__, cmd=self._cmd)
 
-    def __call__(self, job=None):
+    def __call__(self, job):
```

Comment:
This is technically a breaking change. I don't think we use `job=None` anywhere internally though. We do need to change this to work with list of jobs though. Maybe just use `(self, *jobs)`. This allows no jobs to be passed. Then we just do `return self._cmd(*jobs).format(*jobs)`. If we want to catch `None` then we could check the list length.

Reply:
This also may be a good reason to deprecate the call method of `FlowOperation` so as to not need to keep the signature valid between versions.